<ui version="4.0" >
 <class>ImportTagsDialog</class>
 <widget class="QDialog" name="ImportTagsDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Fontmatrix - Tags</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="application.qrc" >
    <normaloff>:/fontmatrix_tags.png</normaloff>:/fontmatrix_tags.png</iconset>
  </property>
  <layout class="QGridLayout" name="gLt" >
   <item rowspan="4" row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="frameShape" >
      <enum>QFrame::Panel</enum>
     </property>
     <property name="text" >
      <string/>
     </property>
     <property name="pixmap" >
      <pixmap resource="application.qrc" >:/fontmatrix_tags_graphic.png</pixmap>
     </property>
     <property name="scaledContents" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2" >
    <widget class="QListWidget" name="tagsList" />
   </item>
   <item row="1" column="1" >
    <widget class="QLineEdit" name="tagText" />
   </item>
   <item row="1" column="2" >
    <widget class="QPushButton" name="tagNewButton" >
     <property name="text" >
      <string>Add</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2" >
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="2" >
    <widget class="QPushButton" name="okButton" >
     <property name="text" >
      <string>Ok</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="application.qrc" />
 </resources>
 <connections/>
</ui>
